packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/CameraPermissions.java

```diff
@@ -23,16 +23,32 @@ interface ResultCallback {
     void onResult(String errorCode, String errorDescription);
   }
 
+  /**
+   * Camera access permission errors handled when camera is created. See {@code MethodChannelCamera}
+   * in {@code camera/camera_platform_interface} for details.
+   */
+  private static final String CAMERA_PERMISSIONS_REQUEST_ONGOING =
```

Comment:
@stuartmorgan Seems like to handle this error in `camera_platform_interface` I need to make a separate PR. Should that PR be landed before this one?